frontend/src/components/formatting/ProjectPageMarkdown.tsx

```diff
@@ -0,0 +1,69 @@
+import { FC } from "react";
+import ReactMarkdown from "react-markdown";
+
+interface ReadmeProps {
+  children: string;
+}
+
+const ProjectPageMarkdown: FC<ReadmeProps> = ({ children }) => {
```

Comment:
If the component config is the same as `Preview`, we can reuse that instead of defining a new component here.

Otherwise, if we anticipate making component changes to the ProjectPageMarkdown that would make it different from Preview, it would be clearer to place this in a `components` folder under the `project_page` folder (e.g. `./frontend/src/components/project_page/components`). Structuring it in this way suggests that the ProjectPageMarkdown is intended for the project page only and not other pages.

Reply:
I moved ProjectPageMarkdown.tsx into the project_page/components directory. I will leave the Preview.tsx component in the formatting directory because it is used in the project preview component and the search results preview component. ProjectPageMarkdown.tsx and Preview.tsx are slightly different and more changes might need to be made to them.